administrate/cron.md

```diff
@@ -136,3 +136,7 @@ fi
 cd ${APP_HOME} # Which has been loaded by the env.
 … # Your part here
 ```
+
+## Standard Output & Error
```

Comment:
Might be better to use a plain `and`. Also, what do you think of `Logs collection` or something like that? Something a bit less technical.